pkg/interfaces/contracts/pool-utils/IRecoveryModeHelper.sol

```diff
@@ -14,15 +14,23 @@
 
 pragma solidity >=0.7.0 <0.9.0;
 
+import "../vault/IVault.sol";
+
 /**
  * Interface for an auxiliary contract that computes Recovery Mode exits, removing logic from the core Pool contract
  * that would otherwise take up a lot of bytecode size at the cost of some slight gas overhead. Since Recovery Mode
  * exits are expected to be highly infrequent (and ideally never occur), this tradeoff makes sense.
  */
 interface IRecoveryModeHelper {
+    /**
+     * @notice Returns the Balancer Vault
+     */
+    function getVault() external view returns (IVault);
+
     /**
      * @dev Computes a Recovery Mode Exit BPT and token amounts for a Pool. Only 'cash' balances are considered, to
-     * avoid scenarios where the last LPs to attempt to exit the Pool cannot because only 'managed' balance remains.
+     * avoid scenarios where the last LPs to attempt to exit the Pool cannot do it because only 'managed' balance
+     * remains.
```

Comment:
The original text is also fine from a pure English perspective. Maybe this is even clearer, since it could still fail with a non-zero, but insufficient, cash balance.

`avoid scenarios where the last attempts to exit the Pool fail with insufficient balance, because some of the total balance requested is 'managed'`